### ISSUE TITLE
Extra whitespace in  SecLists/Discovery/Web_Content/big_portuguese-dictionary.txt

### ISSUE THREAD

**Issue by franciscofreire** [danielmiessler/SecLists#137]
This wordlist seems to have an extra whitespace in the line beginning for each entry. Can you remove these?

**Comment by xXCaulDevsYT** [comment]
Wow

**Comment by g0tmi1k** [comment]
```
[SecLists/Miscellaneous]$ cat lang-portuguese.txt | grep '^ ' | wc -l
41516
[SecLists/Miscellaneous]$ wc -l lang-portuguese.txt
41516 lang-portuguese.txt
[SecLists/Miscellaneous]$ 
```

- - - 

```
$ sed -i '' 's/^ //' lang-portuguese.txt                                                                                                                     $
$ cat lang-portuguese.txt | grep '^ ' | wc -l                                                                                                                      0
$
```

Fixed in #180
